packages/framework/assets/styles/admin/components/box/quick-search.less

```diff
@@ -41,10 +41,8 @@
     &__icon {
         position: absolute;
         right: 15px;
-        top: calc(~"50% - @{quick-search-icon-size}/2");
-        vertical-align: middle;
-
+        top: 50%;
+        transform: translateY(-48%);
```

Comment:
This value is weird, are you sure this is correct?

Reply:
yup, I used the precise measurement tool - E.Y.E. 😄